Tasks/AzureRmWebAppDeployment/ParameterParser.ts

```diff
@@ -0,0 +1,127 @@
+// TODO: This whole file is copied from AzureResourceGroupDeployment task. Move this parser to common lib
+// Similar parser is used in Grid UI extension. Try to move the code to some place where all can use it.
+
+export function parse(input: string) {
```

Comment:
this file is taken from VSO/DistributedTask/Service/WebAccess/DistributedTask/Scripts/Extensions/Common/PowerShellParameters.ts which contains L0 tests so I have not added it here